src/oop/object_independence/medium/q3/QuestionText.md

```diff
@@ -0,0 +1,39 @@
+# oop-object_independence-medium-q3
+
+## Question Info
+
+- Question type: Parsons
+
+## Question Text
+
+Your goal is to become a master chef. So, you make a mobile app to keep track of all your secret recipes. Make a class
+called **Cocktail** with one attribute called **ingredients** which represents a list of ingredients and are initialized
+in the constructor. The method also has an accessor, mutator, and a **toString()** method.
```

Comment:
here also accessor and mutator should be switched to setter and getter for consistency